### DISCUSSION TITLE
[question] Instant Answers

### DISCUSSION THREAD

**Discussion by davidcollini** [searxng/searxng#1173] [Ideas]
I currently use brave search for its instant answers and Discussions feature, is there a way for Searxng to filter instant answers from google, brave or Duckduckgo?

**Comment by mrpaulblack** [comment]
We support instant answers from google at the moment. With that I mean just answers in form of text not the javascript applets like the calculator...

There is an ongoing effort of impl. something like that in the project. But we are in the beginning stages and its going to be a goal for the far away future, sry.

This is the starting point of the discussion about instant answers: https://github.com/searxng/searxng/pull/1022 from there we need to figure out how to impl. this properly in SearXNG which will take some time as the next step.

[EDIT] I also moved this to discussions since we are still in the starting phase...
